packages/devkit/src/integrationmodel.js

```diff
@@ -650,15 +650,26 @@ export default class IntegrationModel {
    * Remove events to formulas in the DOM target.
    */
   removeEvents() {
-    const eventTarget = this.isIframe ? this.target.contentWindow.document : this.target;
+    const eventTarget = this.isIframe && this.target.contentWindow?.document ? this.target.contentWindow.document : this.target;
+    if (!eventTarget) {
+      return;
+    }
     Util.removeElementEvents(eventTarget);
-  }
+    }
```

Comment:
```suggestion
  }
```
Unnecessary added space

Reply:
Fixed, thanks!